setup.py

```diff
@@ -460,7 +459,11 @@ def load(filename):
         _cmake_build_dir, "deps", "magnum-bindings", "src", "python"
     )
 
-    if not args.skip_install_magnum and not is_pip():
+    if (
```

Comment:
Interesting.  When I last had a pip install working pip was every unhappy if you called pip install from a pip install.

Reply:
This was needed in PEP517 mode to make it happy: https://github.com/facebookresearch/habitat-sim/blob/7bdd0b08b8055226eb9b913808282961c01f8f32/pyproject.toml#L6